vncdotool/rfb.py

```diff
@@ -112,6 +116,49 @@
 KEY_SpaceBar=   0x0020
 
 
+# ZRLE helpers
+def _next_bit(it, pixels_in_tile):
```

Comment:
minor nit, should these have shared ```_zrle``` prefix to more easily identity them?

Reply:
done